test/extended/templates/templateinstance_security.go

```diff
@@ -109,14 +109,14 @@ var _ = g.Describe("[templates] templateinstance security tests", func() {
 				},
 			},
 			{
-				by:              "checking adminuser can create a privileged object",
+				by:              "checking adminuser can't create a privileged object",
```

Comment:
this is probably worth a comment that the reason this should fail is because the controller itself should not have these permissions, even though the adminuser does.